tests/functional/pages/surveys/relationship/household-relationship.page.js

```diff
@@ -9,12 +9,14 @@ class HouseholdRelationshipPage extends QuestionPage {
 
   setHusbandOrWifeRelationship(index, relationshipIndex) {
     var id = this.buildRelationshipAnswerId(index, 1)
+    browser.waitForExist(id)
```

Comment:
Yeah think this will be a problem still. When running `while yarn test_functional -- --spec=household-relationship;do; done` to run the command until it failed

I got

```
[chrome #0a] Session ID: 84a64b56-561a-4676-a5c4-d7ff5ff21b3c
[chrome #0a] Spec: /Users/benjefferies/source/github/eq-survey-runner/tests/functional/spec/household-relationship.spec.js
[chrome #0a] Running: chrome
[chrome #0a]
[chrome #0a]   Household relationship
[chrome #0a]       ✓ Given I have added Joe Bloggs, Jane Doe and John Doe to my household when I set Joe Bloggs relationships then I should see the questions `Joe Bloggs is the husband or wife of Jane Doe` and `Joe Bloggs is the son or daughter of John Doe`
[chrome #0a]       ✓ Given I have answered how Joe Bloggs is related Jane Doe and John Doe when I go to the next relationship question then I should see the questions `Jane Doe is the … of John Doe`
[chrome #0a]       ✓ Given I am on the household page when I enter Joe Bloggs then I should not have to enter relationship details
[chrome #0a]       1) Given I answer the relationship questions for Joe Bloggs when answer the relationship questions for Jane Doe then I should not have to enter relationship details for John Doe
[chrome #0a]
[chrome #0a]
[chrome #0a] 3 passing (15s)
[chrome #0a] 1 failing
[chrome #0a]
[chrome #0a] 1) Household relationship Given I answer the relationship questions for Joe Bloggs when answer the relationship questions for Jane Doe then I should not have to enter relationship details for John Doe:
[chrome #0a] unknown error: Element is not clickable at point (372, 613). Other element would receive the click: <label for="who-is-related_1-6" class="label label--inline venus">...</label>
[chrome #0a] Error: unknown error: Element is not clickable at point (372, 613). Other element would receive the click: <label for="who-is-related_1-6" class="label label--inline venus">...</label>
[chrome #0a]     at HouseholdRelationshipPage.setSonOrDaughterRelationship (tests/functional/pages/surveys/relationship/household-relationship.page.js:20:25)
[chrome #0a]     at Context.<anonymous> (tests/functional/spec/household-relationship.spec.js:73:8)
[chrome #0a]     at elementIdClick("53") - click.js:20:22
```